guide/web-streaming.md

```diff
@@ -15,6 +15,19 @@ except that the `X3D` file and the animation are sent on the fly to the Web brow
 
 %end
 
+The web interface displays a toolbar with the following actions:
+* Simulation time: this item indicates the current simulation time on the first line of the simulation element. The second line is currently not used and indicates how long the simulation will run.
```

Comment:
The timeout is not used for broadcast streaming so we cannot display any useful value.
Then if we want to reduce the differences with the simulation streaming server .
Many other features are disabled, so this is only another disabled thing and for me it is not something the user will care about.

But you could suggest any other solution.